src/components/LogButton.js

```diff
@@ -1,26 +1,44 @@
 import React, { Component } from 'react';
-import { Card, Paragraph } from 'react-native-paper';
+import { Button, Paragraph, Title } from 'react-native-paper';
+import { StyleSheet } from 'react-native';
 //REDUX
 import { connect } from 'react-redux';
 import { incrementCount } from '../redux/actions';
+import { hasReachedDailyLimitSelector } from '../redux/selectors';
+
+const styles = StyleSheet.create({
+    habitButton: {
+        padding: 20,
+    },
+    habitButtonText: {
+        display: 'block',
```

Comment:
I'm getting an error when I try to run this - suggests that `block` is not a valid value for `display` here. Did this run locally for you?

Reply:
Yea it ran locally for me with no problems. I’ll revisit to see whats happening

Reply:
Ok - the second time it did not run locally - apologies I must have not run testing after my style update!
You were correct in that styles appeared to work in web browser but not with test or on app. Have removed offending styles and reverted the 'log button' component to a react native paper card component instead of button and added a '+' button to trigger count update. React native paper doesn't support multi-line buttons so rather than run into issues later have made this change now. Tests running locally successfully after updating snapshots.